src/controllers/legacycontrollersettings.cpp

```diff
@@ -17,12 +17,19 @@ namespace {
 const QRegularExpression kHwbtnRe(QStringLiteral(":hwbtn:`([^`]*)`"));
 
 const QString kHwbtnStyleWrapper = QStringLiteral(
-        "<span style='background-color: #111111; "
-        "color: #d9d9d9; "
-        "font-weight: 700;'>"
-        // wrapping the string in &nbsp; is apparently the only way to get a padding
-        // \\1 is the RegEx match group 1
-        "&nbsp;\\1&nbsp;</span>");
+        "<span style='"
+        "background: #343131;"
+        "color: #d9d9d9;"
+        "font-size: 70%;"
+        "font-weight: 600;"
+        "line-height: 120%;"
+        "text-transform: uppercase;"
```

Comment:
I wouldn't enforce that tbh.

Reply:
I'd pick this commit right away, I'd only remove that line
Okay?.

Reply:
> I wouldn't enforce that tbh.

I know we do that in the manual though.
In the end, let's keep it for consistency (instead of changing it in the manual, for each doc...)